docs/cmdline-opts/limit-rate.d

```diff
@@ -17,6 +17,9 @@ Appending 'k' or 'K' will count the number as kilobytes, 'm' or 'M' makes it
 megabytes, while 'g' or 'G' makes it gigabytes. The suffixes (k, M, G, T, P)
 are 1024 based. For example 1k is 1024. Examples: 200K, 3m and 1G.
 
+The rate limiting logic works on averaging the transfer speed to no more than
+the set threshold over a period of multiple seconds.
```

Comment:
Ref: https://github.com/curl/curl/issues/7965

If the goal here is informing the user that rate limiting split second transfers has no effect then I think you should say that, such as "Rate limiting is averaged over several seconds. For shorter transfers it will have no effect."

Reply:
Shorter than what? And it will/might have *some* effect depending on factors. I'm vague to avoid having to make specific claims that I can't back up.

Reply:
If it might have some effect then disregard.

Reply:
This information appears to be outdated. At least the send rate is already limited immediately after sending the header.

Reply:
The limiting logic also appears to have been changed such that it is no longer is based on averaging (see 4b86113f5ed66270364738a351ebb5bb23cd0882, #971)

Reply:
I maintain my added sentence is still correct and might help users understand what to expect from this option.